examples/pygstc/simple_pipeline.py

```diff
@@ -0,0 +1,83 @@
+#The software contained in this file is free and unencumbered software released into the public domain.
+#Anyone is free to use the software contained in this file as they choose, including incorporating
+#it into proprietary software.
+import time
+import sys
+from pygstc.gstc import *
+from pygstc.logger import *
+
+#Create a custom logger with loglevel=DEBUG
+gstd_logger = CustomLogger('simple_pipeline', loglevel='DEBUG')
+
+#Create the client with the logger
+gstc = GstdClient(logger=gstd_logger)
```

Comment:
Can we encapsulate all of this logic in a `Player` class? and have main call methods of this class.